deploy.gradle

```diff
@@ -1,28 +1,28 @@
 remotes {
     dev {
-        host = 'biostudy-bia.ebi.ac.uk'
-        user = 'ma-svc'
+        host = System.getenv("DEV_SERVER")
```

Comment:
Given now we need all to make application work. I think we should document all this in indepent confluence page to avoid losing it all if gitlab is changed or down.

Reply:
https://www.ebi.ac.uk/seqdb/confluence/display/BS/CI+Pipeline